src/signals/incident/containers/KtoContainer/components/KtoForm/index.test.js

```diff
@@ -9,9 +9,8 @@ import ktoMock from '../../../../definitions/kto';
 
 import KtoForm, { andersOptionText } from './index';
 
-
-
-describe('<KtoForm />', () => {
+// temporarily skipping tests for this component; picking up later
+describe.skip('<KtoForm />', () => {
```

Comment:
Do you still want to skip this test?

Reply:
For now, yes. Will tackle it at a later date.